src/test/java/org/broadinstitute/barclay/help/TestExtraDocs.java

```diff
@@ -5,7 +5,7 @@
 /**
  * Class for testing extraDocs property in docgen.
  */
-@DocumentedFeature(groupName = "Test extra docs group name", extraDocs = TestExtraDocs.class)
+@DocumentedFeature(groupName = "Test extra docs group name")
```

Comment:
extraDocs is still tested; its referenced in, and tested by, TestArgumentContainer. I removed it from here because its self-referential, which is harmless but unnecessary.